### PR TITLE
Lessoncolors and Icons Update

### PR DESCRIPTION
**Pull request by OctoDino**
### New Features implemented?
✅  The `frames` in a `lesoncard` will now change `border-color` and `.circle{backgroundColor}` depending on the lesson should be working fine, **i think**
✅  Added Icons as indicators to the frame (see first commit) but they aren't right allinged due to emulator issuess! Please check this! (@FlynnLG)

⚠️ PLEASE CHECK EVERYTHING; EMULATOR ISSUESS ARE STILL EXISTING ON MY MACHINE (coded blind)

### CODE REVIEW

File: evimanager/components/lessoncard.js [hunk 1]
```diff
@@ -40,7 +41,7 @@ export const LessonCard = ({
         </View>
         <View style={styles.weekendHolidayContainer}>
           <View style={styles.weekendHolidayContentInfoContainer}>
-            <Ionicons name="cafe" size={20} color={'#ffffff'} />
+            <Ionicons name="cafe" size={20} color={THEME.fontColor} />
```
Comment (OctoDino) [inline evimanager/components/lessoncard.js:44]: Make Icons fit with the Theme (wouldn't be good with a white background aka Light Theme!

File: evimanager/constants/theme.js [hunk 2]
```diff
@@ -23,3 +25,48 @@ export const FONTS = {
...
+    await EncryptedStorage.setItem('localdata.lessonColors.religion', defaultLessonColors.religion)
+    await EncryptedStorage.setItem('localdata.lessonColors.geo', defaultLessonColors.geo)
+    await EncryptedStorage.setItem('localdata.lessonColors.physik', defaultLessonColors.physik)
+    await EncryptedStorage.setItem('localdata.lessonColors.sport', defaultLessonColors.sport)
+    await EncryptedStorage.setItem('localdata.lessonColors.pb', defaultLessonColors.pb)
+    await EncryptedStorage.setItem('localdata.lessonColors.art', defaultLessonColors.art)
+    await EncryptedStorage.setItem('localdata.lessonColors.ds', defaultLessonColors.ds)
+    await EncryptedStorage.setItem('localdata.lessonColors.music', defaultLessonColors.music)
+    await EncryptedStorage.setItem('localdata.lessonColors.luck', defaultLessonColors.luck)
+    await EncryptedStorage.setItem('localdata.lessonColors.other', defaultLessonColors.other)
```
Comment (OctoDino) [inline evimanager/constants/theme.js:65]: Maybe possible to do this all in a for loop instead like this 💩

File: evimanager/constants/theme.js [hunk 3]
```diff
@@ -23,3 +25,48 @@ export const FONTS = {
...
+    //return default value for the first run
+    return(defaultLessonColors.other)
+  }else{
+    //When it isn't "other" than get thing
+    return(strorageRes)
```
Comment (OctoDino) [inline evimanager/constants/theme.js:70]: returning a string HEX value

File: temp_README.md [hunk 4]
```diff
@@ -142,5 +142,4 @@ MIT
 
 > [amitmerchant.com](https://www.amitmerchant.com) &nbsp;&middot;&nbsp;
 > GitHub [@amitmerchant1990](https://github.com/amitmerchant1990) &nbsp;&middot;&nbsp;
-> Twitter [@amit_merchant](https://twitter.com/amit_merchant)
-
+> Twitter [@amit_merchant](https://twitter.com/amit_merchant)
```
Comment (OctoDino) [inline temp_README.md:145]: deleted on accident

**Review by OctoDino — COMMENTED** [review]
Leaved some comments for better understanding of the code

**Comment by FlynnLG** [comment]
I reviewed all commits and it looks great. Found some small things which can cause problems but all in all it **should** work. Today I'm not able to test this PR in the iOS/Android emulator but tomorrow I will have a look at it and will approve this PR if everything works and if not I will add/remove things by committing to this PR. And when you feel comfortable you can merge the PR into the main branch.
Thanks @OctoDino - although your computer is currently giving you a hard time!

**Pull request closed without merging**